tests/libtest/lib1537.c

```diff
@@ -47,11 +47,11 @@ int test(char *URL)
 
   /* deprecated API */
   ptr = curl_escape((char *)a, asize);
-  printf("%s\n", ptr);
   if(!ptr) {
     res = TEST_ERR_MAJOR_BAD;
     goto test_cleanup;
   }
+  printf("%s\n", ptr);
```

Comment:
Agreed, these two are clearly in the wrong order (even though `ptr` being NULL would cause a test failure as well which is what we want).